openshift/distro/ubi/9/Dockerfile

```diff
@@ -82,6 +79,7 @@ RUN cp -n openshift/3scale_backend.conf /etc/ \
  && gem unpack "apisonator-${BACKEND_VERSION}.gem" --target=/opt/ruby \
  && cd "/opt/ruby/apisonator-${BACKEND_VERSION}" \
  && cp -a ${HOME}/app/.bundle "/opt/ruby/apisonator-${BACKEND_VERSION}/" \
+ && bundle config --local path "/opt/ruby/apisonator-${BACKEND_VERSION}/" \
```

Comment:
This is a bit tricky, apparently bundle gems where installed at `${HOME}/app/.bundle` but the `bundle` command were looking for them at some other location, not sure where. Anyway this fixes the problem.